sdk/tables/azure-data-tables/cgmanifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://json.schemastore.org/component-detection-manifest.json",
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "git",
+        "git": {
+          "RepositoryUrl": "https://github.com/cpplint/cpplint",
+          "CommitHash": "8456631ba3a12842da08238362a90d33fcf47062"
+        }
+      },
+      "DevelopmentDependency": true
+    },
+    {
+      "Component": {
+        "Type": "other",
+        "Other": {
+          "Name": "clang-format",
+          "Version": "9.0.0-2",
+          "DownloadUrl": "https://ubuntu.pkgs.org/18.04/ubuntu-updates-universe-amd64/clang-format-9_9-2~ubuntu18.04.2_amd64.deb.html"
+        }
+      },
+      "DevelopmentDependency": true
+    },
+    {
+      "Component": {
+        "Type": "other",
+        "Other": {
+          "Name": "doxygen",
+          "Version": "1.8.20",
+          "DownloadUrl": "http://doxygen.nl/files/doxygen-1.8.20-setup.exe"
+        }
+      },
+      "DevelopmentDependency": true
+    },
+    {
+      "Component": {
+        "Type": "git",
+        "git": {
+          "RepositoryUrl": "https://gitlab.gnome.org/GNOME/libxml2",
```

Comment:
Why is this here, do you have `libxml2` dependency?

Reply:
some responses/requests are json some are XML . thus we also need xml to deserialize them

Reply:
Where is it used? I don't see a single `#include <libxml/...>`.

In CmakeLists.txt, I don't see `target_link_libraries(... libxml2)`, don't see `find_package()` for it

I don't see it stated as a dependency in `vcpkg/package.json` and `vcpkg/Config.cmake.in`.

Reply:
@antkmsft I believe we want to include transitive dependencies as well, both in cgmanifest.json and in the NOTICE.txt  file, but that's something we need to make a concrete call on based on guidance and put in the guidelines.

https://docs.opensource.microsoft.com/using/guidance-for-open-source-usage/notice-faq/#q-does-the-notice-file-need-to-include-dependencies
> Q: Does the NOTICE file need to include dependencies?
A: Yes, the NOTICE should include the full dependency tree that is [distributed](https://docs.opensource.microsoft.com/legal/resources/GLOSSARY/#distribution) in the [Microsoft Offering](https://docs.opensource.microsoft.com/legal/resources/GLOSSARY/#microsoft-offering). Using CG to generate your NOTICE should automatically include attributions for the full dependency tree.

But also:
https://docs.opensource.microsoft.com/tools/cg/legal/notice/
> In the component list select all components you distribute to customers. The list of components in the list are only direct dependencies. If a direct dependency is included, all transitive dependencies will also be included. 

Given the SDK **does** in fact use an XML parsing library, it's good to keep it for now, and I don't think it's a concern worth "getting right" in this PR, but rather fix-up across the board.

Reply:
Agree